target/linux/ramips/dts/mt7621_xiaomi_mir3g-v2.dts

```diff
@@ -132,9 +133,26 @@
 	};
 };
 
-&ethernet {
-	mtd-mac-address = <&factory 0xe000>;
-	mediatek,portmap = "llllw";
+&gmac0 {
+	mtd-mac-address = <&factory 0xe006>;
+};
+
+&switch0 {
+	ports {
+		port@2 {
+			status = "okay";
+			label = "lan2";
+		};
+		port@3 {
+			status = "okay";
+			label = "lan1";
+		};
+		port@4 {
+			status = "okay";
+			label = "wan";
+			mtd-mac-address = <&factory 0xe000>;
```

Comment:
wan_mac=0xe006!
So, gmac0 should be kept at 0xe000, and label-mac-device should be changed to wan.